### PR TITLE
COS Resizable

### PR DESCRIPTION
**Pull request by JamesIves**
Makes the COS wheel resizable via a prop value. 

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)

### CODE REVIEW

File: components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx [hunk 1]
```diff
@@ -242,7 +249,7 @@ const ChanceOfSuccess = (props) => {
             </g>
           )}
           <text
-            fill={isDark ? colors.midnight : colors.white}
+            fill={colors.white}
```
Comment (RohitRajendran) [inline components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx:252]: How come you're getting rid of the isDark?

File: components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx [hunk 2]
```diff
@@ -323,8 +331,19 @@ ChanceOfSuccess.propTypes = {
   compact: PropTypes.bool,
   /** Adjusts the appearance of the component if the user is using Internet Explorer. */
   isIe: PropTypes.bool,
-  /** Renders the ChanceOfSuccess meter to appear more kindly on light backgrounds. */
+  /** Renders the ChanceOfSuccess meter to appear more kindly on light backgrounds. */	  /** The height of the meter. */
```
Comment (RohitRajendran) [inline components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx]: Typo

File: components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx (hunk 1 above)
Comment (JamesIves) [reply components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx]: oopsie

File: components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx (hunk 2 above)
Comment (JamesIves) [reply components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx]: also oopsie

File: components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx [hunk 3]
```diff
@@ -323,8 +331,19 @@ ChanceOfSuccess.propTypes = {
...
+  /** The height of the meter. */
+  height: PropTypes.string,
+  /** The width of the meter. */
+  width: PropTypes.string,
+  /** Additional style properties to apply to the container. */
+  style: PropTypes.objectOf(PropTypes.string),
+};
+
+ChanceOfSuccess.defaultProps = {
+  height: '14.1rem',
```
Comment (RohitRajendran) [inline components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx:345]: Is the addition of these default props going to affect and current uses of this component?

Comment (JamesIves) [reply components/atoms/ChanceOfSuccess/ChanceOfSuccess.jsx:345]: I made the default props match what the default height used to be without a height/width value. It was hardcoded as `141px` in the stylesheet (which I've now removed)

**Pull request merged**